.github/actions/slack-file/message_file_to_slack.sh

```diff
@@ -58,7 +58,7 @@ fi
 
 if [ -e "${PAYLOAD_FILE}" ]; then
     jq -n \
-    --arg msg "$(cat "${PAYLOAD_FILE}")" \
+    --arg msg "$(head -n 50 < "${PAYLOAD_FILE}")" \
```

Comment:
This looks fine by itself.  

Depending on how the caller job that generate this $PAYLOAD_FILE, we may run into a case where we cannot look at the full log.  For example, the caller job pipes the stdout and stderr to $PAYLOAD_FILE.  

I would suggest piping the stderr to stdout and teeing it to a file in the caller job. something like `2&>1 | tee ${PAYLOAD_FILE}`

Reply:
Just saw @junkil-park  comment about this in slack on this as well.  I really dislike taking ownership of the job for them - no analysis of why the job misbehaved was attempted before asking me for help - I was going to push back before realizing I could fix their job in the action.   Anyway, I'll also open a pr to fix their build step for them.   I wont print large messages in the action as it'll just encourage future bad logging in jobs.

Reply:
@rexhoffman, I am sorry for the miscommunication. I never intended for you to take full ownership of our job. I didn't mean for you to fix our problem entirely on your own. I was simply asking for your insight on the issue or guidance on where I could start fixing the issue. I am more than willing to maintain our part. Thank you for your help. I appreciate it and again, I apologize for the miscommunication.